src/cartridges/int_adyen_overlay/cartridge/scripts/handleCustomObject.ds

```diff
@@ -162,6 +161,16 @@ function handle( customObj : CustomObject  ) : Object
 				pending = true;
 				Logger.getLogger("Adyen", "adyen").info ("Order {0} was in pending status.",order.orderNo);
 				break;				
+		  case "CAPTURE":
+				if(customObj.custom.success !== "true" && order.status === Order.ORDER_STATUS_CANCELLED ){
```

Comment:
This also needs to be customObj.custom.success == "true"

Reply:
indeed